internal/cluster/transition.go

```diff
@@ -176,7 +176,11 @@ func (th *transitionHandler) PostResetCluster(sw stateswitch.StateSwitch, args s
 		}
 	}
 
-	return th.updateTransitionCluster(params.ctx, logutil.FromContext(params.ctx, th.log), params.db, sCluster, params.reason, extra...)
+	err := th.updateTransitionCluster(params.ctx, logutil.FromContext(params.ctx, th.log), params.db, sCluster, params.reason, extra...)
```

Comment:
Can we do resetLastInstallationPreparationStatus as part of updateTransitionCluster with extra ?

Reply:
Makes sense, I will change